community/modules/scheduler/cloud-batch-job/outputs.tf

```diff
@@ -14,11 +14,23 @@
  * limitations under the License.
  */
 
+locals {
+  provided_instance_tpl_msg  = "An instance template was provided to be used by your Batch job"
+  generated_instance_tpl_msg = "An instance template was created for your Batch job"
+}
+
 output "instructions" {
-  description = "Instructions for submitting Google Cloud Batch job."
+  description = "Instructions for submitting the Batch job."
   value       = <<-EOT
+
+  A Batch job template file has been created locally at:
+    ${abspath(local.job_template_output_path)}
+
+  ${var.instance_template == null ? local.generated_instance_tpl_msg : local.provided_instance_tpl_msg} & referenced in the above file:
```

Comment:
The `&` seems a bit out of place in the code and in the output. I'd just use "and".

Reply:
In fact, overall I would simplify to:

```
The Batch job template uses the existing instance template:
```

or

```
The Batch job template uses a new instance template created matching the provided settings:
```

Reply:
Done.